plone/app/upgrade/v51/profiles.zcml

```diff
@@ -121,4 +121,14 @@
       for="Products.CMFPlone.interfaces.IMigratingPloneSiteRoot"
       provides="Products.GenericSetup.interfaces.EXTENSION"
       />
+
+  <genericsetup:registerProfile
+      name="to518"
+      title="Upgrade profile for Plone 5.1.7 to Plone 5.1.8"
+      description=""
+      directory="profiles/to_518"
+      for="Products.CMFPlone.interfaces.IMigratingPloneSiteRoot"
+      provides="Products.GenericSetup.interfaces.EXTENSION"
+      />
```

Comment:
This directory doen't exist anymore ... you can remove this change

Reply:
that's right

Reply:
Needs to be removed

Reply:
It's done